src/dataset.py

```diff
@@ -139,6 +142,7 @@ def name_lookup(
             elif dataset_select == Datasets.seattle:
                 htmls = []
                 for r in records:
+                    date = datetime.strptime(r.get("date"), "%Y-%m-%dT%H:%M:%SZ")
```

Comment:
Honestly I think this could just be a `date object with `%Y-%m-%d` format, the time of day isn't going to mean much :smile:

Reply:
This is because the datetime format from the SQL database is POSIX or ISO or something or other and has all the H:M:SZ junk on the end, so we've gotta specify those parts here for the conversion.

Reply:
Oh oh right, okay I misunderstood this line when I first read it. Sorry about that!